.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -182,9 +182,6 @@ stages:
             docker run --rm \
             -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
             -w /go/src/github.com/Azure/AgentBaker \
-            -e CLIENT_ID=${AZURE_BUILD_CLIENT_ID} \
```

Comment:
are you giving packer an MSI now instead? if so how are you accomplishing this?

I've wanted to get this working for our builds for a long time now, but my understanding is that packer doesn't support cross-tenant managed identities

Reply:
Yes. I'm giving packer an MSI now. Referring to https://developer.hashicorp.com/packer/integrations/hashicorp/azure#managed-identity, I remove the `client_id`, `client_secret` and `tenant_id`. So packer will attempt to use the Managed Identity and subscription of the VM that Packer is running on, which is related to our 1es hosted pool.